<launch>

    <node pkg="car_control" type="car_control" name="car_control_node" output="screen">
    <!-- <remap from="cmd_vel" to="cmd_vel_twist"/> -->
        <param name="sub_topic" value="cmd_vel" />
        <param name="pub_topic" value="ctrl_cmd" />
        <param name="speed_radio" value="1.0" />
        <param name="rot_radio" value="1.0" />
        <param name="max_speed" value="0.3" />
        <param name="max_rot_speed" value="30" />
    </node>

</launch>


